README.md

```diff
@@ -92,6 +92,17 @@ book].
 
 This library makes use of the [ndk-rs](https://github.com/rust-windowing/android-ndk-rs) crates, refer to that repo for more documentation.
 
+The `ndk_glue` version needs to match the version used by `winit`. Otherwise, the application will not start correctly as `ndk_glue`'s internal NativeActivity static is not the same due to version mismatch.
+
+`ndk_glue` <-> `winit` version comparison tables:
```

Comment:
Yes! I actually prefer like that as well :)

And yes, I have tested all combinations from `winit >= "0.24` in android. And since I have a personal project going on to run on Desktop (macOS), iOS and Android, I will keep an eye to this for the foreseeable future.